server/src/main/java/io/spine/server/aggregate/AggregateRepository.java

```diff
@@ -104,10 +114,14 @@
     @Nullable
     private Set<CommandClass> commandClasses;
 
-    /** The set of event classes to which aggregates of this repository {@linkplain React react}. */
+    /** The set of event classes on which aggregates {@linkplain React react}. */
     @Nullable
     private Set<EventClass> reactedEventClasses;
 
+    /** The set of rejection classes on which aggregates {@linkplain React react}. */
+    @Nullable
+    private Set<RejectionClass> reactedRejectionClasses;
```

Comment:
If that's an expected set of classes, not the actual (i.e. this field is not a statement of a fact, but rather a declaration of what's expected), then I would name it like `expectedRejectionReactions` or `declaredRejectionReactions`. Similarly, the set of event-related reactions should be renamed as well.

The bottom line is that I'd like to avoid the feeling I get from this kind of naming. Right now it looks like a piece of history.